### ISSUE TITLE
Text in Understanding 3.3.2 allows something functionally identical to HTML placeholder

### ISSUE THREAD

**Issue by dd8** [w3c/wcag#1136]
The HTML `placeholder` attribute does not satisfy SC 3.3.2 - this is explored in some detail in
https://github.com/w3c/wcag/issues/864

However, this line in "Understanding Success Criterion 3.3.2: Labels or Instructions" describes something that appears functionally identical to a placeholder, and has the same problems outlined in issue 864:

> A field for entering a date contains initial text which indicates the correct format for the date.

https://www.w3.org/WAI/WCAG21/Understanding/labels-or-instructions.html

**Comment by patrickhlauke** [comment]
yup, although one could argue there's a bit of nuance there potentially (that the initial text which indicates..." is meant as ancilliary information, and it's implied there's still a label, etc), i agree that even ancilliary info must remain visible to satisfy 3.3.2. would suggest removing/rewording this, yes.

**Comment by dd8** [comment]
There's also this example:

> In a form which contains both required and optional fields, the required fields and/or the optional fields are clearly labeled as such.

This could be used to argue that merely labelling a required field with an asterisk is enough to pass this SC.

**Comment by patrickhlauke** [comment]
not *every* input requires instructions though, hence the "*or*" (otherwise we'd end up with forms becoming overly verbose if authors were required normatively to have labels *and* instructions).

regarding the asterisk as only indicator: purely for a non-AT user, i'd say that's "fine" due to the long-established convention that this is used for required fields? i'd personally pass this for 3.3.2 (though would suggest to authors to at least add a "all fields marked with * are required" or, better, use actual text like "(required)" (or conversely, "(optional)" when the majority is required and only a handful are optional). but those would be best practice recommendations, rather than requirements to pass 3.3.2, in my view.
